scripts/build_coverage.sh

```diff
@@ -172,8 +172,13 @@ if [ "$skip_gn" == false ]; then
     # 3) Run Python tests if requested
     #
     if [ "$ENABLE_PYTHON" == true ]; then
+        echo "Building python and applications ..."
+        scripts/run_in_build_env.sh \
+            "./scripts/tests/local.py build"
```

Comment:
does this build with coverage?

Reply:
Yes, it is triggered by './scripts/tests/local.py build'